src/styles/utilities/display.js

```diff
@@ -20,4 +20,8 @@ export default {
     dInline: {
         display: 'inline',
     },
+
+    dGrid: {
+        display: 'grid',
+    },
```

Comment:
Remove this no need to add it here. Since it would create some unnecessary confusion